### PR TITLE
Update Galaxy 101 tutorial

### PR DESCRIPTION
**Pull request by nsoranzo**
- Update the "Galaxy 101" tutorial to use the `hg38` genome and the latest dbSNP version from the UCSC Genome Browser. Also update the images to show the UI of Galaxy 17.05.

- A couple of small fixes for the for peaks2genes tutorial.

### CODE REVIEW

File: topics/introduction/tutorials/galaxy-intro-101/metadata.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ objectives:
 - "Learn how to share your work"
 time_estimation: "1-1.5h"
 key_points:
-- "Galaxy provides an easy-to-use graphical user interface for often complex commandline tools"
+- "Galaxy provides an easy-to-use graphical user interface for often complex command-line tools"
```
Comment (bebatut) [inline topics/introduction/tutorials/galaxy-intro-101/metadata.yaml]: Can you rather modify the main `metadata.yaml` file in `introduction` directory? It is the one that is used...

File: topics/introduction/tutorials/galaxy-intro-101/tutorial.md [hunk 2]
```diff
@@ -23,32 +23,32 @@ This practical aims to familiarize you with the Galaxy user interface. It will t
...
-> *Mom (or Dad) ... Which coding exon has the highest number of single nucleotide polymorphisms (SNPs) on human chromosome 22?*
+> *Which coding exon has the highest number of single nucleotide polymorphisms (SNPs) on human chromosome 22?*
 {: .question}
 
 You are thinking "Wow! This is a simple question... I know where to find the data, at the [UCSC Genome Browser](https://genome.ucsc.edu/), but how do I actually compute this?" There is really a straightforward way of answering this question and it is called **Galaxy**. So let's try it...
 
-Browse to your Galaxy instance and log in or register. The Galaxy interface consists of three main parts. The available tools are listed on the left, your analysis history is recorded on the right, and the middle pane will show the home page, the selected tool or the dataset.
+Browse to your Galaxy instance and log in or register. The Galaxy interface consists of three main parts. The available tools are listed on the left, your analysis history is recorded on the right, and the middle pane will show the home page, a tool form or some dataset content.
 
 ![](../../images/galaxy_interface.png)
```
Comment (bebatut) [inline topics/introduction/tutorials/galaxy-intro-101/tutorial.md]: Can you add an `alt` text for the images, if possible? And do you know that you can add a legend for the images by adding it right after the path to the image (in the curly brackets)?

File: topics/introduction/tutorials/galaxy-intro-101/metadata.yaml (hunk 1 above)
Comment (nsoranzo) [reply topics/introduction/tutorials/galaxy-intro-101/metadata.yaml]: Sorry, this is a bit confusing. Should the redundant metadata be removed from this file? If yes, what should be left here?

File: topics/introduction/tutorials/galaxy-intro-101/tutorial.md (hunk 2 above)
Comment (nsoranzo) [reply topics/introduction/tutorials/galaxy-intro-101/tutorial.md]: > And do you know that you can add a legend for the images by adding it right after the path to the image (in the curly brackets)?

You suppose you mean the round brackets.

Comment (willdurand) [reply topics/introduction/tutorials/galaxy-intro-101/tutorial.md]: Yes, inside parentheses

File: topics/introduction/tutorials/galaxy-intro-101/metadata.yaml (hunk 1 above)
Comment (bebatut) [reply topics/introduction/tutorials/galaxy-intro-101/metadata.yaml]: This file is supposed to be removed (PR in progress 😉). Sorry for the confusion

**Comment by nsoranzo** [comment]
@bebatut I think I've addressed your review comments, thanks!

**Comment by bebatut** [comment]
@nsoranzo Thanks for this PR 😄

**Pull request merged**